packages/yourloops/pages/login/login.tsx

```diff
@@ -115,7 +142,9 @@ function Login(props: RouteComponentProps): JSX.Element {
     } catch (reason: unknown) {
       log.error(reason);
       setValidateError(true);
-      const message = _.isError(reason) ? reason.message : new String(reason).toString();
+      const message = _.isError(reason)
```

Comment:
I have add an utility function `errorTextFromException()` in `packages/yourloops/lib/utils.ts` if you want.